Rakefile

```diff
@@ -1,3 +1,22 @@
-require 'puppet/version'
+require 'rake'
 require 'rspec/core/rake_task'
 require 'puppetlabs_spec_helper/rake_tasks'
+begin
+  require 'beaker/tasks/test' unless RUBY_PLATFORM =~ /win32/
```

Comment:
Ah, I see it in registry... but note that registry is really old, and might not be the best model to follow.

https://github.com/puppetlabs/puppetlabs-registry/blob/master/Rakefile#L3-L7